### DISCUSSION TITLE
关于插件的名字

### DISCUSSION THREAD

**Discussion by morytyann** [nikkinikki-org/OpenWrt-nikki#107] [General]
感觉是时候起个名字了，以下是Owner想到的备选项

1. OriginProxy
2. ZeroProxy
3. CircleProxy
4. YATP（Yet Another Transparent Proxy）

你可以回复数字进行投票。
或者如果你想到了其他的好名字，欢迎回复，我会及时添加到备选项里供大家选择。

**Comment by liumingan** [comment]
一定要选，我就选最简单的，字母最少的

**Comment by zutzo** [comment]
4

**Comment by include-nan** [comment]
或者参照隔壁，叫 openMihomo ?

**Reply by morytyann** [reply]
这个肯定不行，个人不喜欢蹭热度，而且我已经改了

**Comment by morytyann** [comment]
已经修改为`Mihomo for OpenWrt`，类似`Clash.Meta for Android`，缩写为MFOW
